.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -1,140 +1,146 @@
+#        command: ".buildkite/scripts/win_unit_tests.ps1"
+#        notify:
+#          - github_commit_status:
+#              context: "Filebeat: windows/Unit Tests"
+#        agents:
+#          provider: "gcp"
+#          image: "{{matrix.image}}"
+#          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+#          disk_size: 200
+#          disk_type: "pd-ssd"
+#        matrix:
+#          setup:
+#            image:
+#              - "${IMAGE_WIN_2016}"
+#              - "${IMAGE_WIN_2022}"
+#        artifact_paths:
+#          - "filebeat/build/*.xml"
+#          - "filebeat/build/*.json"
+#
+#  - group: "Extended Testing"
+#    key: "extended-tests"
+#    if: build.env("BUILDKITE_PULL_REQUEST") != "false" || build.env("GITHUB_PR_TRIGGER_COMMENT") == "filebeat for extended support"
+#
+#    steps:
+#      - label: ":linux: ARM64 Unit Tests"
+#        key: "arm-extended"
+#        if: build.env("GITHUB_PR_TRIGGER_COMMENT") == "filebeat for arm" || build.env("GITHUB_PR_LABELS") =~ /.*arm.*/
+#        command:
+#          - ".buildkite/filebeat/scripts/unit-tests.sh"
+#        notify:
+#          - github_commit_status:
+#              context: "Filebeat/Extended: Unit Tests ARM"
+#        agents:
+#          provider: "aws"
+#          imagePrefix: "${AWS_IMAGE_UBUNTU_ARM_64}"
+#          instanceType: "${AWS_ARM_INSTANCE_TYPE}"
+#        artifact_paths: "filebeat/build/*.xml"
+#
+#      - label: ":mac: MacOS Unit Tests"
+#        key: "macos-extended"
+#        if: build.env("GITHUB_PR_TRIGGER_COMMENT") == "filebeat for macos" || build.env("GITHUB_PR_LABELS") =~ /.*macOS.*/
+#        command:
+#          - ".buildkite/filebeat/scripts/unit-tests.sh"
+#        notify:
+#          - github_commit_status:
+#              context: "Filebeat/Extended: MacOS Unit Tests"
+#        agents:
+#          provider: "orka"
+#          imagePrefix: "${IMAGE_MACOS_X86_64}"
+#        artifact_paths: "filebeat/build/*.xml"
+#
+#  - group: "Windows Extended Testing"
+#    key: "extended-tests-win"
+#    if: build.env("GITHUB_PR_TRIGGER_COMMENT") == "filebeat for windows" || build.env("GITHUB_PR_LABELS") =~ /.*windows.*/
+#
+#    steps:
+#      - label: ":windows: Win 2019 Unit Tests"
+#        key: "win-extended-2019"
+#        command: ".buildkite/scripts/win_unit_tests.ps1"
+#        notify:
+#          - github_commit_status:
+#              context: "Filebeat/Extended: Win-2019 Unit Tests"
+#        agents:
+#          provider: "gcp"
+#          image: "${IMAGE_WIN_2019}"
+#          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+#          disk_size: 200
+#          disk_type: "pd-ssd"
+#        artifact_paths:
+#          - "filebeat/build/*.xml"
+#          - "filebeat/build/*.json"
 
   - group: "Packaging"
     key: "packaging"
     if: build.env("BUILDKITE_PULL_REQUEST") != "false"
-    depends_on:
-      - "mandatory-tests"
+#    depends_on:
+#      - "mandatory-tests"
 
     steps:
       - label: Package pipeline
```

Comment:
lets use the ingest image as well, because it will speed up a lot the checkout, due to the ref-repo